contracts/Bridge.sol

```diff
@@ -348,6 +351,7 @@ contract Bridge is Pausable, Context {
     /**
         @notice It's used to trigger the belonging process on the MPC side.
         It's used to trigger the belonging process on the MPC side which also handles keygen function calls order.
+        @notice Only callable by addresses that have function call access.
```

Comment:
What do you mean by: `that have function call access.`

Reply:
That the caller is an address mapped in `functionAccess` - has right to call that function

Reply:
So maybe we should write `@notice Only callable by addresses mapped in functionAccess` or something like that? Because for me it is unclear what this sentence mean right now